1748739761450	Sync.LogManager	DEBUG	Flushing file log
1748739761479	Sync.LogManager	DEBUG	Log cleanup threshold time: 1747875761479
1748739761483	Sync.LogManager	DEBUG	Done deleting files.
1748743361518	Sync.Service	DEBUG	User-Agent: Firefox/139.0 (Linux x86_64) FxSync/1.141.0.20250522210034.desktop
1748743361518	Sync.Service	INFO	Starting sync at 2025-06-01 08:02:41 in browser session dVKA5ttehGyA
1748743361518	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1748743361520	Sync.Status	INFO	Resetting Status.
1748743361520	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1748743361522	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1748743361522	FirefoxAccounts	DEBUG	getOAuthToken enter
1748743361522	FirefoxAccounts	TRACE	getCachedToken returning cached token
1748743361522	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1748743361523	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1748743361524	Sync.SyncAuthManager	INFO	Getting sync key
1748743361524	FirefoxAccounts	DEBUG	getOAuthToken enter
1748743361524	FirefoxAccounts	TRACE	getCachedToken returning cached token
1748743361524	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1748743361524	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1748743361525	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1748743361525	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1748743361525	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1748743361528	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1748743361528	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1748743361529	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1748743361529	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1748743361530	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1748743361530	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1748743361530	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1748743361530	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1748743361532	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1748743361532	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1748743361535	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/264067527/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1748743361535	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1748743361535	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1748743361535	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1748743361535	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1748743361536	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1748743361539	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
